src/components/navigation/navigation-item/navigation-item.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation,
+  ContentChildren,
+  QueryList,
+  Renderer2,
+  ElementRef,
+  Optional,
+  SkipSelf,
+  AfterContentInit,

```

Comment:
Review imports, some of these are never used

Reply:
Removed unused imports.